frontend/src/pages/org/archived-item-detail/ui/qa.ts

```diff
@@ -505,20 +474,199 @@ export class ArchivedItemDetailQA extends TailwindElement {
+              const latestFinishedSelected =
+                this.qaRunId === finishedQARuns[0].id;
+
+              return html`
+                <div>
+                  <sl-tooltip
+                    content=${mostRecentSelected
+                      ? msg(
+                          "You’re viewing the latest results from a finished analysis run.",
+                        )
+                      : msg(
+                          "You’re viewing results from an older analysis run.",
+                        )}
+                  >
+                    <sl-tag
+                      size="small"
+                      variant=${mostRecentSelected ? "success" : "warning"}
+                    >
+                      ${mostRecentSelected
+                        ? msg("Current")
+                        : latestFinishedSelected
+                          ? msg("Last Finished")
+                          : msg("Outdated")}
+                    </sl-tag>
+                  </sl-tooltip>
+                  <btrix-qa-run-dropdown
+                    .items=${finishedQARuns}
+                    selectedId=${this.qaRunId || ""}
+                    @btrix-select=${(e: CustomEvent<SelectDetail>) =>
+                      (this.qaRunId = e.detail.item.id)}
+                  ></btrix-qa-run-dropdown>
+                </div>
+              `;
+            })}
+          </div>
+          <div class="flex items-center gap-2 text-neutral-500">
+            ${when(
+              qaRun.state.startsWith("stop") ||
+                (qaRun.state === "complete" &&
+                  qaRun.stats.done < qaRun.stats.found),
+              () =>
+                html`<sl-tooltip
+                  content=${qaRun.state.startsWith("stop")
+                    ? msg("This analysis run was stopped and is not complete.")
+                    : msg(
+                        "Not all pages in this crawl were analyzed. This is likely because some pages are not HTML pages, but other types of documents.",
+                      )}
+                  class="[--max-width:theme(spacing.56)]"
+                >
+                  <sl-icon
+                    name="exclamation-triangle-fill"
+                    class="text-warning"
+                    label=${msg("Note about page counts")}
+                  ></sl-icon>
+                </sl-tooltip> `,
+            )}
+            ${when(
+              qaRun.stats,
+              (stats) => html`
+                <div class="text-sm font-normal">
+                  ${formatNumber(stats.done)} / ${formatNumber(stats.found)}
+                  ${pluralOf("pages", stats.found)} ${msg("analyzed")}
+                </div>
+              `,
+            )}
+            <sl-tooltip
+              content=${msg(
+                "Match analysis compares pages during a crawl vs. during an analysis run. A good match indicates that the crawl is probably good, whereas severe inconsistencies may indicate a bad crawl.",
```

Comment:
```suggestion
                "Match analysis compares pages during a crawl vs. their replay during an analysis run. A good match indicates that the crawl is probably good, whereas severe inconsistencies may indicate a bad crawl.",
```

Might be a little clearer?